substrate/frame/support/procedural/src/runtime/parse/mod.rs

```diff
@@ -187,6 +187,13 @@ impl Def {
 				}
 			}
 
+			if pallet_index.is_none() {
```

Comment:
```suggestion
			let Some(pallet_index) = pallet_index else {
```

Reply:
`pallet_index` can be None for all items other than type aliases.